src/flapjackfeeder.c

```diff
@@ -473,29 +541,37 @@ char *expand_escapes(const char* src)
 }
 
 int generate_event(char *buffer, size_t buffer_size, char *host_name, char *service_name,
-                   char *state, char *output, char *long_output, int event_time) {
+                   char *state, char *output, char *long_output, char *tags,
+                   long initial_failure_delay, long repeat_failure_delay, 
+                   int event_time) {
 
-    char *escaped_host_name           = expand_escapes(host_name);
-    char *escaped_service_name        = expand_escapes(service_name);
-    char *escaped_state               = expand_escapes(state);
-    char *escaped_output              = expand_escapes(output);
-    char *escaped_long_output         = expand_escapes(long_output);
+    char *escaped_host_name    = expand_escapes(host_name);
+    char *escaped_service_name = expand_escapes(service_name);
+    char *escaped_state        = expand_escapes(state);
+    char *escaped_output       = expand_escapes(output);
+    char *escaped_long_output  = expand_escapes(long_output);
 
     int written = snprintf(buffer, buffer_size,
                             "{"
-                                "\"entity\":\"%s\","    // HOSTNAME
-                                "\"check\":\"%s\","     // SERVICENAME
-                                "\"type\":\"service\"," // type
-                                "\"state\":\"%s\","     // HOSTSTATE
-                                "\"summary\":\"%s\","   // HOSTOUTPUT
-                                "\"details\":\"%s\","   // HOSTlongoutput
-                                "\"time\":%d"           // TIMET
+                                "\"entity\":\"%s\","                   // HOSTNAME
+                                "\"check\":\"%s\","                    // SERVICENAME
+                                "\"type\":\"service\","                // type
+                                "\"state\":\"%s\","                    // HOSTSTATE
+                                "\"summary\":\"%s\","                  // HOSTOUTPUT
+                                "\"details\":\"%s\","                  // HOSTlongoutput
+                                "\"tags\":[%s],"                       // tags
+                                "\"initial_failure_delay\":%lu,"       // initial_failure_delay
+                                "\"repeat_failure_delay\":%lu,"        // repeat_failure_delay
+                                "\"time\":%d"                          // TIMET
                             "}",
                                 escaped_host_name,
                                 escaped_service_name,
                                 escaped_state,
                                 escaped_output,
                                 escaped_long_output,
+                                tags,
```

Comment:
Oops, my test this time was wrong. I end up with escaped json and of course flapjack throws an error on parsing that.
Reverted.